tests/integration/test_sanity.py

```diff
@@ -21,7 +21,6 @@ def test_info_before_cluster_init(cluster):
     node = RedisRaft(1, cluster.base_port, cluster.config)
     node.start()
     assert node.info()["raft_state"] == "uninitialized"
-    assert node.client.execute_command("info")["module"]["name"] == "raft"
```

Comment:
redis-py intercepts some commands and parse them, info is one of them. I think this is why map syntax works with those square brackets. 

See the PR, they changed the behavior: https://github.com/redis/redis-py/pull/1393
because there might be multiple lines of "module" s.